server/database/migrations/20190529131154-create-users.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Users', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      firstName: {
+        type: Sequelize.STRING
+      },
+      lastName: {
+        type: Sequelize.STRING
+      },
+      email: {
+        type: Sequelize.STRING
+      },
+      password: {
+        type: Sequelize.STRING
+      },
+      avatarUrl: {
+        type: Sequelize.STRING
+      },
+      verified: {
+        type: Sequelize.BOOLEAN
+      },
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      },
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      }
```

Comment:
Please remove the 'createdAt' and 'updatedAt' properties from the migration file. If someone tries to run migrations with this file it will create those columns in the table and the person won't be able to seed the database without putting values for those fields.

Reply:
By removing this, does that not mean, we do not have the date and time a user details was added or updated.

Reply:
I checked the 'users' file in the models folder. If those properties are not there in the users file then there's no point for them to be there in the migration file. Secondly, the database diagram for users does not contain those fields so there's no point for it to be there. Besides those fields were generated by Sequelize by default. The bottom line is that if it's not contained in the database diagram then it should not be there. If we decide to add it, then we should as well add it to all the tables that are going to be created, and I don't think that's a good idea.

Reply:
Noted, would confirm from @KvNGCzA , whether createdAt and updatedAt should be added to the database design or removed from the generated default file for users